pkg/render/kube-controllers.go

```diff
@@ -156,6 +156,15 @@ func (c *kubeControllersComponent) controllersRole() *rbacv1.ClusterRole {
 		}
 
 		role.Rules = append(role.Rules, extraRules...)
+
+		if c.cr.Spec.ClusterManagementType == operator.ClusterManagementTypeManagement {
+			// For cross-cluster requests an authentication review will be done for authenticating the kube-controllers.
+			role.Rules = append(role.Rules, rbacv1.PolicyRule{
+				APIGroups: []string{"projectcalico.org"},
+				Resources: []string{"authenticationreviews"},
+				Verbs:     []string{"create"},
+			})
+		}
```

Comment:
Would be good to add some test coverage for these config changes.

Reply:
Added